internal/server/metrics.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/stream-tester/internal/metrics"
+)
+
+type MetricsServer struct {
+}
+
+// NewStreamerServer creates new MetricsServer
+func NewMetricsServer() *MetricsServer {
+	return &MetricsServer{}
+}
+
+// Starts metric server
```

Comment:
```suggestion
// Start starts metric server
```

Reply:
I think the correction is actually a typo. Do you mind if I keep the original?

Reply:
`golint internal/server/metrics.go`